examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -457,6 +480,9 @@ function zero_moment_microphysics_tendency!(Yₜ, Y, p, t)
                 (1 - ᶜλ) * TD.internal_energy_ice(params, ᶜts)
             )
     end
+
+    # update precip in cache for coupler's use
+    col_integrated_precip = vertical∫_col(ᶜS_ρq_tot) ./ FT(1e3) # ρ_water = FT(1e3)
```

Comment:
Could we use the [water density](https://github.com/CliMA/CLIMAParameters.jl/blob/a60edbad941c13aacac60ba0b16998802f52421d/src/Planet/planet_parameters.jl#L12) in climaparameters here?